<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CustomListWidget</class>
 <widget class="QWidget" name="CustomListWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>680</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>680</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>CustomListWidget</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget#CustomListWidget
{
  background-color:#000000;
}

QWidget#headerWidget
{
  background-color:#0C1114;
  border-left: 1px solid gray;
  border-right: 1px solid gray;
  border-top: 0px solid gray;
  border-bottom: 0px solid gray;
}

QLabel{
 color:#FFFFFF
}

QListView{
  border-left: 1px solid gray;
  border-right: 1px solid gray;
  border-top: 0px solid #172025;
  border-bottom: 1px solid gray;
  background-color:#172025;
}

QScrollBar:vertical{
  background-color:black;
  width:8px;
}
QScrollBar::handle:vertical{
   background: white;
   min-height: 60px;
}

QScrollBar::up-arrow:vertical, QScrollBar::down-arrow:vertical {
     border: 0px solid grey;
     width: 0px;
     height: 0px;
 }
</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="headerWidget" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>40</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>40</height>
      </size>
     </property>
     <widget class="QLabel" name="labelSerisId">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>36</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>序列号</string>
      </property>
      <property name="wordWrap">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QLabel" name="labelStatus">
      <property name="geometry">
       <rect>
        <x>202</x>
        <y>0</y>
        <width>24</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>状态</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelTime">
      <property name="geometry">
       <rect>
        <x>403</x>
        <y>0</y>
        <width>72</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>标注完成时间</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelOperation">
      <property name="geometry">
       <rect>
        <x>605</x>
        <y>0</y>
        <width>24</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>操作</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelImportTime">
      <property name="geometry">
       <rect>
        <x>500</x>
        <y>0</y>
        <width>72</width>
        <height>15</height>
       </rect>
      </property>
      <property name="text">
       <string>导入时间</string>
      </property>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QListView" name="listView">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>600</height>
      </size>
     </property>
     <property name="autoFillBackground">
      <bool>true</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">QListView::item{
 height:60px;
}</string>
     </property>
     <property name="lineWidth">
      <number>0</number>
     </property>
     <property name="modelColumn">
      <number>0</number>
     </property>
     <property name="batchSize">
      <number>50</number>
     </property>
     <property name="selectionRectVisible">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="navWidget" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>100</height>
      </size>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="navLayout">
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
